jubakit/loader/core.py

```diff
@@ -16,17 +16,12 @@ def __init__(self, f, close=False):
     self._close = close
     self._lines = 0
 
-  def __iter__(self):
-    try:
-      for line in self._f:
-        yield self.preprocess({str(self._lines): line})
-        self._lines += 1
-    finally:
-      if not self._f.closed and self._close:
-        self._f.close()
-
-  def preprocess(self, ent):
-    return {'line': list(ent.values())[0]}
+  def rows(self):
+    for line in self._f:
```

Comment:
``try ~ finally`` is unnecessary?
I think the modification of #10 was removed in rebase process.